Lib/tkinter/test/test_ttk/test_style.py

```diff
@@ -23,11 +34,48 @@ def test_configure(self):
 
     def test_map(self):
         style = self.style
-        style.map('TButton', background=[('active', 'background', 'blue')])
-        self.assertEqual(style.map('TButton', 'background'),
-            [('active', 'background', 'blue')] if self.wantobjects else
-            [('active background', 'blue')])
-        self.assertIsInstance(style.map('TButton'), dict)
+
+        # Single state
+        for states in ['active'], [('active',)]:
+            style.map('TButton', background=[('active', 'white')])
```

Comment:
The loop variable `states` is not used in the loop, making it irrelevant.  By analogy with the next 2 loops, did you mean
```suggestion
            style.map('TButton', background=[(*states, 'white')])
```
?